.nvmrc

```diff
@@ -0,0 +1 @@
+16.15.1
```

Comment:
TBD
Node version supported by AWS is 16.15.0. I think we are ok using 16.15.1, there aren't many differences between them.

https://docs.aws.amazon.com/elasticbeanstalk/latest/platforms/platform-history-nodejs.html